docs/sidebar.yml

```diff
@@ -9,17 +9,13 @@ subsection:
         subsection:
           - page: reference/new-types/intersection-types.md
           - page: reference/new-types/intersection-types-spec.md
-            hidden: true
```

Comment:
What is the motivation for this change?

Reply:
We found it confusing that you could get into the hidden docs by mistake when you clicked "Next page". If you did, you moved to a page that was not in the nav list on the left. I think that the final solution should be to make some special kind of navigation link that would allow you to go to the details page, and to keep the `Next` pointing to the next page that is not hidden.

Reply:
I agree with the observation, but I don’t think this is the best way to address it. I think the “Next” buttons should always skip hidden pages.